lib/util/exec/types.ts

```diff
@@ -35,6 +35,7 @@ export interface RawExecOptions extends ChildProcessSpawnOptions {
    */
   encoding: string;
   maxBuffer?: number | undefined;
+  cwd?: string | undefined;
```

Comment:
```suggestion
  cwd?: string;
```

`?` already adds `undefined` type 😉

Reply:
we should fix our types, as it should never undefined

https://github.com/renovatebot/renovate/blob/f8c789efb44311d74320865c8cb85abc015b73c4/lib/util/exec/index.ts#L55-L61

I'll do it in seperate PR